lib/foods.js

```diff
@@ -46,7 +46,18 @@ function clearErrors(){
 }
 
 function addDeleteFunctionality(){
-  $("td.food-delete button").on('click', function(){
+  $("#food-list").on('click', "button", function(){
+    var row = $(this).parentsUntil("tbody")
+    var item = row[1].firstElementChild.innerHTML
+    var foodItems = JSON.parse(localStorage.getItem("food-items"))
+  for (var i = 0; i < foodItems.length; i++) {
```

Comment:
This could be a `$.grep` function that checks each element to make sure it is not equal to the `foodItem` you're looking at. This looks like works though!